helm/aqua-app-server/templates/gate-deployment.yaml

```diff
@@ -33,6 +33,10 @@ spec:
           value: "{{ .Release.Name }}-console-svc:443"
         - name: AQUA_WORKLOADS_INTEGRITY_CHECK
           value: {{ .Values.gate.integrityCheck | default "true" | quote }}
+        - name: AQUA_WORLOADS_PING_CHECK
+          value: {{ .Values.gate.pingCheck | default "true" | quote }}
+        - name: AQUA_DISCONNECTION_GRACE_PERIOD
+          value: {{ .Values.gate.disconnectionGracePeriod | default "true" | quote }}
```

Comment:
is this actually a boolean variable? if it isn't then defaulting to true could break stuff.

Reply:
it's a info we got from them

Reply:
> Gateway - AQUA_DISCONNECTION_GRACE_PERIOD (boolean; true by default)